### ISSUE TITLE
CodeLens feature breaks on type cast

### ISSUE THREAD

**Issue by h4de5** [firsttris/vscode-jest-runner#164]
CodeLens feature (run, debug) stops working for the whole file, when a type casting (`<type>`) is used somewhere it seems.

casting using `as type` still works.

![2021-04-26_130631_J6pd](https://user-images.githubusercontent.com/6115324/116073166-5666a100-a690-11eb-974e-4d9aab98cba6.gif)

```typescript
// breaks
describe('TestCodeLens', () => {
  it('should be true', () => {
    expect( <number> 1).toBeTruthy();
  });
});
```

```typescript
// works
describe('TestCodeLens', () => {
  it('should be true', () => {
    expect( 1 as number ).toBeTruthy();
  });
});
```

using:
vscode 1.55.2
jest runner v0.4.34

**Comment by ehaynes99** [comment]
This is caused by this issue: https://github.com/jest-community/jest-editor-support/issues/68

**Comment by gregoryzh** [comment]
> This is caused by this issue: [jest-community/jest-editor-support#68](https://github.com/jest-community/jest-editor-support/issues/68)

created PR for that, hopefully we could land it soon and resolve this.

**Comment by firsttris** [comment]
should be fixed in v0.4.35 please re-open if the issue persists
